src/web/mjs/connectors/OremangaNet.mjs

```diff
@@ -0,0 +1,53 @@
+import OremangaNet from '../engine/OremangaNet.mjs';
```

Comment:
I explained to you on the previous PR that you are importing a non existing file, and yet you do the same.

What are you trying to do? 

You need to import "Connector" from "Connector.mjs". Or, maybe, since this website template is the same a Neumanga  something like : "import Neumanga from ./Neumanga.mjs" ?

Reply:
since this website template is the same a Neumanga
Can you recommend me?

Reply:
And, How can I test the program?

Reply:
Personally i work in in \AppData\Local\hakuneko-desktop\cache\mjs (modifying my file) and i launch Hakuneko. I was to lazy to setup NodeJS at the time i started.

Reply:
> since this website template is the same a Neumanga Can you recommend me?

i told you what to do already ^^